dapr/clients/exceptions.py

```diff
@@ -38,3 +41,72 @@ def as_dict(self):
             'errorCode': self._error_code,
             'raw_response_bytes': self._raw_response_bytes,
         }
+
+
+class DaprGrpcError(RpcError):
+    def __init__(self, err: RpcError):
```

Comment:
Please add a `json()` method which can print a map of all the error attributes and their values.

That would be most convenient for just dumping the entire error in a more readable manner.